examples/byob/main.tf

```diff
@@ -34,7 +34,7 @@ module "resources" {
 resource "google_storage_bucket_iam_member" "admin" {
   bucket = module.resources.bucket_name
   member = "serviceAccount:${var.service_account_email}"
-  role   = "roles/storage.admin"
+  role   = "roles/storage.objectAdmin"
```

Comment:
I thought the service account needs `storage.admin` permissions to create the bucket?

Reply:
The account running this tf needs storage.admin to create the account. We need storage.objectAdmin to access the bucket.